app/store/models/User.d.ts

```diff
@@ -126,12 +126,14 @@ export type PublicUserWithGroups = Pick<
 > &
   PublicUser;
 
-export type AdministrateUser = Pick<User, 'abakusGroups' | 'allergies'> &
-  PublicUser;
+export type AdministrateUser = Pick<User, 'abakusGroups'> & PublicUser;
 
 export type AdministrateExportUser = Pick<User, 'email' | 'phoneNumber'> &
   AdministrateUser;
 
+export type AdministrateaAllergiesUser = Pick<User, 'allergies'> &
```

Comment:
I dont really know I just figured it was necessary as a part of receiving the data from backend

Reply:
For the entire Allergies route I have taken a lot of inspiration from the Attendees route which has functionality for exportation that is similar to what I wanted to accomplish. Therefore I though maybe this would be useful as well. 

After looking more into it I notice that the AdministrateExportUser type is only used in the a UnknownUser type, which is only used in the Entities interface (app/store/models/Entities.ts). 
I thought that maybe The AdministrateAllergiesUser could be meaningful to also have in the UnknownUser type since the export type is there, however i dont know what any of that does. 

If you think its unnecessary I will remove it.

Reply:
Added it to the unknown user type